js/main.js

```diff
@@ -27,6 +27,7 @@ var pinTemplate = document.querySelector('#pin')
 var adForm = document.querySelector('.ad-form');
 var fieldDisabled = document.querySelectorAll('fieldset');
 var mapFilters = document.querySelector('.map__filters');
+var blockSelect = mapFilters.querySelectorAll('select');
```

Comment:
```suggestion
var blockSelects = mapFilters.querySelectorAll('select');
```
https://up.htmlacademy.ru/javascript/19/criteries#b5